doc/user/flycheck-versus-flymake.rst

```diff
@@ -44,71 +37,68 @@ Please do **not** use this table alone to make your personal judgment.  Read the
-|syntax checkers        |function/macro         |various variables [#]_ |
-+-----------------------+-----------------------+-----------------------+
-|Functions as syntax    |yes                    |**no** [#]_            |
-|checkers               |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-|Error levels           |errors, warnings,      |errors, warnings       |
-|                       |informational, custom  |                       |
-|                       |levels                 |                       |
-+-----------------------+-----------------------+-----------------------+
-|Error identifiers      |yes                    |no                     |
-+-----------------------+-----------------------+-----------------------+
-|Error parsing          |Regular expressions,   |Regular expressions    |
-|                       |custom parsers for     |                       |
-|                       |structured formats     |                       |
-|                       |(XML, JSON, etc.)      |                       |
-+-----------------------+-----------------------+-----------------------+
-|Multiline error        |yes                    |**no**                 |
-|messages               |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-|Error highlighting in  |yes                    |yes                    |
-|buffers                |                       |                       |
-+-----------------------+-----------------------+-----------------------+
-|Fringe icons for errors|yes                    |yes (Emacs 24.1+)      |
-+-----------------------+-----------------------+-----------------------+
-|Error message display  |Tooltip, echo area,    |Tooltip only [#]_      |
-|                       |fully customizable     |                       |
-+-----------------------+-----------------------+-----------------------+
-|List of all errors     |yes                    |**no**                 |
-+-----------------------+-----------------------+-----------------------+
-|Resource consumption   |low                    |**high**               |
-+-----------------------+-----------------------+-----------------------+
-|Unit tests             |all syntax checkers,   |**none?**              |
-|                       |large parts of         |                       |
-|                       |internals              |                       |
-+-----------------------+-----------------------+-----------------------+
++---------------------------+-----------------------+-----------------------+
+|                           |Flycheck               |Flymake                |
++===========================+=======================+=======================+
+|Supports Emacs versions    ||min-emacs|            |22+                    |
```

Comment:
This is wrong, it only supports Emacs 26

Reply:
Indeed!